src/main/play-doc/operation/Install.md

```diff
@@ -990,60 +729,189 @@ Proxying application endpoints is required when running more than one instance o
 
-Setup basic HAProxy configuration for the Docker based HAProxy.
+Once the ConductR `.tgz` is obtained, extract the ConductR-HAProxy bundle from the package in a working directory, we'll use `workdir`. The ConductR HAProxy bundle can be found within the ConductR package under  `conductr-{version}/extra/conductr-haproxy-{version}-{digest}.zip`.
+
+
+### Use CLI to load and run ConductR-HAProxy bundle
+
+Upload the ConductR-HAProxy bundle to the bastion host and use the [[CLI|CLI]] to load the ConductR-HAProxy bundle.
 
 ```bash
-[172.17.0.1]$ sudo mkdir /etc/haproxy
-[172.17.0.1]$ sudo touch /etc/haproxy/haproxy.cfg
+$ conduct load file://conductr-haproxy-{version}-{digest}.zip
 ```
 
-If the agent is executing as a named user, enable the user to own the configuration file
+Scale ConductR-HAProxy so that ConductR-HAProxy is running on every proxy node in the cluster. In our case we have 3 nodes where the proxy is expected to be running, so we scale up the ConductR-HAProxy to 3 instances.
 
 ```bash
-[172.17.0.1]$ sudo chown conductr-agent:conductr-agent /etc/haproxy/haproxy.cfg
+$ conduct run conductr-haproxy --scale 3
+```
+
+## Verifying if bundle has been started successfully
+
+Use the command `conduct info` from the bastion host to verify successful startup of the bundle.
+
+```bash
+$ conduct info
+ID       NAME        #REP  #STR  #RUN
+6e68f05  visualizer     1     0     1
+```
+
+In the example above, a bundle called `visualizer` has been started successfully.
+
+This can also be confirmed by executing the `dcos task` command which will display the bundle running as a task from the context of DC/OS. The `visualizer-1` task belongs to the currently running `visualizer` bundle having `1` as the `compatibilityVersion`.
+
+```bash
+$ dcos task
+NAME           HOST       USER  STATE  ID
+visualizer-1   10.0.3.75  root    R    6e68f055d1f5715ad3ff19172fa5efaf_0f6e119c-9288-425a-89e3-36379dcaccda
+```
+
+The `compatibilityVersion` is explained in the to [bundle configuration](BundleConfiguration).
+
+The bundle can now be scaled to run multiple instances.
+
+```bash
+$ conduct run --scale=3 visual
+```
+
+Test service endpoints from the public nodes to ensure connectivity. The `/etc/haproxy/haproxy.cfg` file on the public slaves contains the frontend and backend configurations for services exposed by bundles. Test the front end bind port and path using `curl` and the public node ip address to ensure proxying is working correctly.  
+
+
+## Configuring the service gateway
+
+Now that we have configured ConductR-HAProxy to run on the public nodes, we'll want to configure them as a public service. If the public nodes have public IP addresses, the addresses can be listed in DNS directly. For better resiliency, use multiple public proxy nodes.
+
```

Comment:
ELB/Proxy info is still useful here. The 'for example ELB' version probably works well enough here.